x-pack/plugins/reporting/public/management/__test__/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ReportApiJSON } from '../../../common/types';
+
+export interface PayloadMock {
+  payload: Omit<ReportApiJSON['payload'], 'browserTimezone' | 'version' | 'layout'>;
+}
+export type ReportMock = Omit<
```

Comment:
Since there is nothing specific to `public/management` here, maybe just have this code in `reporting/common`?